web/tests/test_chapters.py

```
@@ -0,0 +1,634 @@
+
+class ChapterApplicationTests(TestCase):
+    def setUp(self) -> None:
+        self.client = Client()
+        
+        self.user = User.objects.create_user(
+            username="testuser", email="test@example.com", password="password123",
+        )
+
+    def test_only_verified_members_can_apply(self) -> None:
+        # Create a non-verified user
+        non_verified = User.objects.create_user(
```

Ruff (0.8.2)

Local variable `non_verified` is assigned to but never used

Remove assignment to unused variable `non_verified`

(F841)